### PR TITLE
v0.1.5 Load associated token accounts by owner

### PR DESCRIPTION
**Pull request by Tristyn**

### CODE REVIEW

File: libraries/ts/src/token/associatedToken.ts [hunk 1]
```diff
@@ -91,61 +91,100 @@ export class AssociatedToken {
...
 
     return await this.loadMultipleAux(connection, addresses, decimals)
   }
 
-  /** Loads multiple token accounts. If the native mint is provided, loads the native SOL balance of the owner instead. */
+  /**
+   * Loads multiple associated token accounts by owner.
+   * If the native mint is provided, loads the native SOL balance of the owner instead.
+   * If a mints array is not provided, loads all associated token accounts and the SOL balance of the owner. */
   static async loadMultipleOrNative(
```
Comment (jetjake) [inline libraries/ts/src/token/associatedToken.ts:109]: This is a pretty big method, can we break it up to more modular pieces?

File: libraries/ts/src/token/associatedToken.ts [hunk 2]
```diff
@@ -160,19 +199,41 @@ export class AssociatedToken {
...
   ): Promise<AssociatedToken[]> {
     if (Array.isArray(decimals) && decimals.length !== addresses.length) {
       throw new Error("Decimals array length does not equal addresses array length")
     }
 
     const pubkeys = addresses.map(address => translateAddress(address))
 
-    const accounts = await connection.getMultipleAccountsInfo(pubkeys)
-    return accounts.map((account, i) => {
+    const accountInfos = await connection.getMultipleAccountsInfo(pubkeys)
```
Comment (jetjake) [inline libraries/ts/src/token/associatedToken.ts:210]: this looks similar to the above code block.  Maybe good to create a modular function to reduce code duplication.

**Review by jetjake — CHANGES_REQUESTED** [review]
Requesting a few changes.  Feel free to argue against my positions

**Comment by Tristyn** [comment]
I was thinking the same thing. I can address it soon

**Review by jetjake — APPROVED** [review]
Talked to Tristyn, we will merge this and fix later.

**Pull request merged**